<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VipStatusItem</class>
 <widget class="QWidget" name="VipStatusItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>137</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>123</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>137</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget {
    background-color: transparent;
}
QLabel {
    color: #b0bec5;
    font: 11px &quot;Open Sans&quot;;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(55, 71, 79, 255), stop:0.5 rgba(38, 50, 56, 255), stop:1 rgba(34, 42, 48, 255));
border: none;
border-bottom: 1px solid #191919;
border-left: 1px solid #34424a;
border-right: 1px solid #191919;
padding-top: -4px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="11,0" columnstretch="520,118,79,119,119">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" alignment="Qt::AlignBottom">
    <widget class="QLabel" name="header_description">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>@header_description</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignBottom">
    <widget class="QLabel" name="header_neededRake">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>@header_neededRake</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="header_rakeback">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>@header_rakeback</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="header_statusPrizes">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>@header_statusPrizes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="header_vipTournaments">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>@header_vipTournaments</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="value_description">
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: transparent;
    border: 1px solid #34424a;
    border-right: 1px solid #191919;
    border-bottom: none;
 background-color: #222a30;
border-top-right-radius: 1px 0px;
padding: 12px 20px 16px 30px;
font: 13px &quot;Open Sans&quot;;
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="value_neededRake">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: transparent;
    border: 1px solid #34424a;
    border-right: 1px solid #191919;
    border-bottom: none;
 background-color: #222a30;
border-top-right-radius: 1px 0px;
padding: 3px;
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="value_rakeback">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: transparent;
    border: 1px solid #34424a;
    border-right: 1px solid #191919;
    border-bottom: none;
 background-color: #222a30;
border-top-right-radius: 1px 0px;
padding: 3px;
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="value_statusPrizes">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: transparent;
    border: 1px solid #34424a;
    border-right: 1px solid #191919;
    border-bottom: none;
 background-color: #222a30;
border-top-right-radius: 1px 0px;
padding: 3px;
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="value_vipTournaments">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: transparent;
    border: 1px solid #34424a;
    border-right: 1px solid #191919;
    border-bottom: none;
 background-color: #222a30;
border-top-right-radius: 1px 0px;
padding: 3px;
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
